packages/framework/attributor/src/test/attribution/documents/default/attributor-lz4-and-delta-snap.json

```diff
@@ -10,7 +10,7 @@
op2\":\"v1BgG\",\"prop1\":\"m4xRB\",\"prop3\":\"8e12Q\"},\"type\":2},{\"pos1\":31,\"pos2\":32,\"props\":{\"prop2\":\"v1BgG\",\"prop1\":\"m4xRB\",\"prop3\":\"8e12Q\"},\"type\":2},{\"pos1\":31,\"pos2\":33,\"props\":{\"prop2\":\"v1BgG\",\"prop1\":\"m4xRB\",\"prop3\":\"8e12Q\"},\"type\":2},{\"pos1\":31,\"pos2\":32,\"props\":{\"prop2\":\"v1BgG\",\"prop1\":\"m4xRB\",\"prop3\":\"8e12Q\"},\"type\":2},{\"pos1\":31,\"pos2\":32,\"props\":{\"prop2\":\"v1BgG\",\"prop1\":\"m4xRB\",\"prop3\":\"8e12Q\"},\"type\":2},{\"pos1\":31,\"pos2\":33,\"props\":{\"prop2\":\"v1BgG\",\"prop1\":\"m4xRB\",\"prop3\":\"8e12Q\"},\"type\":2}],\"type\":3},\"referenceSequenceNumber\":79,\"type\":\"op\",\"sequenceNumber\":80,\"minimumSequenceNumber\":76},{\"clientId\":\"a2df62bb-461f-4c2e-a3ad-fc5df463c464\",\"clientSequenceNumber\":28,\"contents\":{\"pos1\":31,\"pos2\":36,\"type\":1},\"referenceSequenceNumber\":80,\"type\":\"op\",\"sequenceNumber\":81,\"minimumSequenceNumber\":76},{\"clientId\":\"d3fdea22-7906-496f-9fc5-0bf988954452\",\"clientSequenceNumber\":28,\"contents\":{\"ops\":[{\"pos1\":30,\"pos2\":40,\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"},\"type\":2},{\"pos1\":31,\"pos2\":35,\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"},\"type\":2},{\"pos1\":31,\"pos2\":33,\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"},\"type\":2},{\"pos1\":31,\"pos2\":34,\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"},\"type\":2},{\"pos1\":31,\"pos2\":34,\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"},\"type\":2},{\"pos1\":31,\"pos2\":32,\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"},\"type\":2},{\"pos1\":31,\"pos2\":34,\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"},\"type\":2},{\"pos1\":31,\"pos2\":33,\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"},\"type\":2},{\"pos1\":31,\"pos2\":38,\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"},\"type\":2},{\"pos1\":31,\"pos2\":40,\"props\":{\"prop2\":\"dHWn1\",\"prop3\":\"5KoWp\"},\"type\":2}],\"type\":3},\"referenceSequenceNumber\":81,\"type\":\"op\",\"sequenceNumber\":82,\"minimumSequenceNumber\":76},{\"clientId\":\"d3fdea22-7906-496f-9fc5-0bf988954452\",\"clientSequenceNumber\":29,\"contents\":{\"pos1\":31,\"seg\":\"yc5Bz\",\"type\":0},\"referenceSequenceNumber\":82,\"type\":\"op\",\"sequenceNumber\":83,\"minimumSequenceNumber\":76},{\"clientId\":\"d3fdea22-7906-496f-9fc5-0bf988954452\",\"clientSequenceNumber\":30,\"contents\":{\"pos1\":36,\"seg\":\"mQny\",\"type\":0},\"referenceSequenceNumber\":83,\"type\":\"op\",\"sequenceNumber\":84,\"minimumSequenceNumber\":76},{\"clientId\":\"25adaa92-9220-49e2-a852-3c7e97ac9d1c\",\"clientSequenceNumber\":24,\"contents\":{\"ops\":[],\"type\":3},\"referenceSequenceNumber\":84,\"type\":\"op\",\"sequenceNumber\":85,\"minimumSequenceNumber\":76},{\"clientId\":\"a2df62bb-461f-4c2e-a3ad-fc5df463c464\",\"clientSequenceNumber\":29,\"contents\":{\"pos1\":4,\"seg\":\"x2NAdmR\",\"type\":0},\"referenceSequenceNumber\":85,\"type\":\"op\",\"sequenceNumber\":86,\"minimumSequenceNumber\":76}]"
```

Comment:
Per @Abe27342 this change to what's output here is benign.

https://github.com/microsoft/FluidFramework/pull/19242/files#r1479094891